src/components/NavMenu.js

```diff
@@ -30,7 +30,7 @@ class NavMenu extends Component {
         path={e.slug}
         children={({ match }) => (
           <li >
-            <Link className={match ? 'active' : ''} to={e.slug}>{e.name}</Link>
+            <Link className={match ? 'active' : ''} to={e.items[0].slug}>{e.name}</Link>
```

Comment:
the change to navigation matches what we discussed, it might have made sense to do 2 separate commits for tracking purposes, but probably not a big deal

Reply:
I'd be happy to split the change in NavMenu.js out but that will be a one-liner and the behavior will not be quite complete. When you first login, the first tab will not be underlined (because it needs the change in login.js). Clicking around on other menus after that will give you the right behavior. This will be temporary until the rest of this change merges anyhow, so it shouldn't be too much of an issue. If you're ok with that, I'll make the change.